app/views/curation_concerns/base/_relationships_collection_row.html.erb

```diff
@@ -0,0 +1,16 @@
+<tr>
+<th><%= t(".#{model_name}", type: model_name) %></th>
+<td>
+  <% if collection.blank? %>
+    <p><%= t('.empty', type: model_name) %></p>
+  <% else %>
+    <ul class="tabular">
+      <% collection.each do |col| %>
+        <li class='attribute title'>
+          <%= link_to col.title.first, contextual_path(col, nil) %>
```

Comment:
This line might render a child work, parent work, or a Collection.. Collections don't have the parent presenter as far as I know. I've done some debugging and also tried using polymorphic_path([col]) in this case and it doesn't work properly. What am I missing, can you suggest a better way to render this link?